ogr/services/github/check_run.py

```diff
@@ -168,14 +171,19 @@ def completed_at(self) -> Optional[datetime.datetime]:
         return self.raw_check_run.completed_at
 
     @property
-    def output(self) -> GithubCheckRunOutput:
+    def output(self) -> CheckRunOutput:
```

Comment:
what was there was just type alias, so it should be fine, not even sure whether the thing provided by PyGithub is some custom data type _that can be used as dictionary_ or just the dictionary :thinking: